Automattic/jetpack
# VideoPress: Disable button for adding video when user cannot upload videos

Fixes #26604

#### Changes proposed in this Pull Request:

* Checks if the user can upload videos before enabling `Add new video` button

#### Other information:


#### Jetpack product discussion


N/A

#### Does this pull request change what data or activity we track or use?


No.

#### Testing instructions:






* Without a paid plan, go to the VideoPress dashboard
* Check that the button is enabled
* Upload a video
* Check that the button is disabled
* Add a paid VideoPress plan
* Notice the button is re-enabled

Without paid plan:
![2022-10-04_18-01-04](https://user-images.githubusercontent.com/8486249/193930332-dd1301f1-181f-4a36-af5e-68229a31025d.png)
![2022-10-05_10-18-12](https://user-images.githubusercontent.com/8486249/194070390-ff997772-13f9-48e1-a036-f36e6e125538.png)

With paid plan:
![2022-10-04_18-13-54](https://user-images.githubusercontent.com/8486249/193930552-27baaa8c-377f-4412-a715-b6a7afb2c263.png)
